_episodes/23-continuous-integration-automated-testing.md

```diff
@@ -94,8 +94,8 @@ like this:
 
 ~~~
 first_scaled_by:
-  - Hans Meyer
-  - Ludwig Purtscheller
+- Hans Meyer
```

Comment:
```suggestion
- Hans Meyer
```

Reply:
We have checked with online YAML linting tool https://www.yamllint.com/ and it prefers the indentation as was originally done in the material. I do not mind either way as both are correct - it's just the readability that matters in this case. Maybe @steve-crouch can weight in on this one?

Reply:
@aleks On balance I'd say go with two spaces of indentation as it is, since it appears to be the normal convention in this case (well, either 2 or four, but some claim 4 spaces can decrease readability in the event of long lines and wrapping)

Reply:
Thanks for both your thoughts - I guess I'm a bit conflicted!

Making it consistent with yamllint makes sense, and I agree this format is easier to read, but having it be different from the GitHub actions later on (when that is the purpose of the YAML introduction) feels like it could trip someone up. 

We could reformat the GH Actions, but that _also_ seems like a bad idea, as then it will be different from the templates GitHub provide.

I'm happy to drop this commit if that's what everyone else thinks is best 🙂

Reply:
OK, let's then go with the formatting used by GH Actions then - that does make sense and will be used later on in the course and both formatting are correct and we are not breaking anything.

Reply:
Great - from my perspective please merge whenever :)